lib/org/chromium/webidl/CanonicalizerRunner.js

```diff
@@ -28,14 +29,36 @@ foam.CLASS({
       name: 'source',
     },
     {
+      class: 'String',
+      name: 'ioRelationshipType',
+      documentation: 'The n:m relationship type of input-to-output.',
+      value: 'many:1',
+    },
+    {
+      class: 'Class',
+      documentation: 'Type of input parameter of run().',
+      name: 'inputType',
+      factory: function() {
```

Comment:
I believe that `foam.core.Class` can adapt strings. `value: 'org.chromium.webidl.ast.Definition'` should be fine here.